server/src/main/java/com/google/musicanalysis/util/Secrets.java

```diff
@@ -0,0 +1,36 @@
+package com.google.musicanalysis.util;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+
+public final class Secrets {
+  private static Object SYNC_OBJECT = new Object();
+  private static SecretManagerServiceClient CLIENT;
```

Comment:
This isn't technically a constant (it's a singleton). It should use lowerCamelCase.